cesium/writer_open.go

```diff
@@ -27,18 +27,40 @@ import (
 
 // WriterMode sets the operating mode of the writer, optionally enabling or disabling
 // persistence and streaming.
+// WriterMode is an uint8, its right-most bit represents the value of Persist(),
+// second-to-right-most bit represents the value of Stream(), and the third-to-right-most
+// bit represents the value of AutoCommit().
 type WriterMode uint8
```

Comment:
You need to make sure you let the rest of the team know when you make breaking changes to the public API of Cesium like this. This change to writer mode has pretty dramatic implications all the way upstream to the C++, TS, Python libraries as well as the Server.

Reply:
If we're going to go the route of using bit masks here, why don't we just make them separate booleans in the config instead? I know you use bitmasks for file modes, but IMO it's confusing. Using booleans is way simpler.